src/openrct2/management/NewsItem.cpp

```diff
@@ -484,8 +484,7 @@ void News::RemoveItem(int32_t index)
     gameState.NewsItems[newsBoundary - 1].Type = News::ItemType::Null;
 }
 
-void News::importNewsItems(
-    GameState_t& gameState, const std::vector<News::Item>& recent, const std::vector<News::Item>& archived)
+void News::importNewsItems(GameState_t& gameState, const std::span<News::Item> recent, const std::span<News::Item> archived)
```

Comment:
```suggestion
void News::importNewsItems(GameState_t& gameState, const std::span<const News::Item> recent, const std::span<const News::Item> archived)
```
Since we are not planning to mutate the input we have to make the element const, making std::span const just means we can't re-assign std::span but News::Item remains mutable.